Website/Packages/datax-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "datax-common",
-  "version": "1.1.1",
+  "version": "1.2.2-SNAPSHOT-5-21",
```

Comment:
shouldn't we remove the SNAPSHOT from the version string?

Reply:
here and the others below

Reply:
in master we plan to add snapshot. the version in stable branch wont have snapshot

Reply:
+1, why do we have a snapshot in version name?